mods/beds/api.lua

```diff
@@ -91,8 +91,9 @@ function beds.register_bed(name, def)
 			destruct_bed(pos, 1)
 		end,
 
-		on_rightclick = function(pos, node, clicker)
+		on_rightclick = function(pos, node, clicker, itemstack, pointed_thing)
```

Comment:
Although these 5 variables are the full list of variables that can be passed it is acceptabe to leave out as many as you want. Here you are passing unnecessary variables that are not used, making the code slower and more complex for no reason. In fact 'node' can be removed as it is not used.